packages/firestore/src/core/bundle.ts

```diff
@@ -265,3 +249,77 @@ export class BundleLoader {
     return new BundleLoadResult({ ...this.progress }, changedDocuments);
   }
 }
+
+/**
+ * Loads a Firestore bundle into the SDK. The returned promise resolves when
+ * the bundle finished loading.
+ *
+ * @param bundleReader Bundle to load into the SDK.
+ * @param task LoadBundleTask used to update the loading progress to public API.
+ */
+export function syncEngineLoadBundle(
```

Comment:
This should live in SyncEngine as it uses the private SyncEngine types (SyncEngineImpl should not be used outside of sync_engine.ts)

Reply:
Done.